code/modules/shuttle/manipulator.dm

```diff
@@ -23,6 +23,14 @@
 /obj/machinery/shuttle_manipulator/Initialize()
 	. = ..()
 	update_icon()
+	SSshuttle.manipulator = src
+
+/obj/machinery/shuttle_manipulator/Destroy(force)
```

Comment:
Maybe out of scope but why the hell is this thing a machinery?

Reply:
Because when I made the manipulator system, I thought it would be cool to have the actual world changing machinery in Centcom?

But these days, it's a machinery because no one bothered changing it.

Reply:
Point and laugh at a man making another blackbox.

Reply:
it's not like it's a hidden aspect of the shuttle system, it got through all the maintainer review and no one complained. So really, point and laugh at the maintainers for not pointing out that something was bad.

Reply:
UUUH This
https://github.com/tgstation/tgstation/blob/master/code/game/machinery/_machinery.dm#L120
This machinery will be in two systems processing FNR when it only should be in one. Please override this.